stages/01_download_data.py

```diff
@@ -0,0 +1,18 @@
+# Imports
+# -------
+import os
+import urllib.request
+
+os.makedirs('downloads', exist_ok=True)
+
+data_links = {
```

Comment:
With a given download list, it would be better to keep the names as they are by just taking the last component of the URI path. This keeps the data closer to the original (which helps if the source ever changes) and there will be no cognitive load of looking up how this brick maps the file names.